### ISSUE TITLE
CGFontGetGlyphPath deprecated

### ISSUE THREAD

**Issue by felixjung** [dcjones/Gadfly.jl#472]
Hi,

this message snuck into my Julia console when doing some plotting on my Mac, today.
```
Oct 24 09:54:23 [...] julia[41321] <Error>: The function ‘CGFontGetGlyphPaths’ is obsolete
and will be removed in an upcoming update. Unfortunately, this application, or a library it uses,
is using this obsolete function, and is thereby contributing to an overall degradation of system
performance.
```

Looks like this is a function provided by Cairo/Quartz?

@dcjones are you aware of this?

Thanks,

Felix

**Comment by ryandesign** [comment]
It was fixed in cairo 1.14.2.

**Comment by aviks** [comment]
Thanks @ryandesign .
